app/uk/gov/hmrc/agentclientrelationships/connectors/EnrolmentStoreProxyConnector.scala

```diff
@@ -67,7 +67,8 @@ class EnrolmentStoreProxyConnector @Inject()(
         else {
           if (userIds.lengthCompare(1) > 0)
             Logger(getClass).warn(s"Multiple userIds found for $enrolmentKeyPrefix")
-          userIds.head
+          //using .head here could get admin or assistant, only want admin
```

Comment:
i think we can remove this comment all together as going forward we see the  mention of `.head` here :)

Reply:
👍